src/components/application_manager/include/application_manager/application_impl.h

```diff
@@ -136,6 +136,18 @@ class ApplicationImpl : public virtual Application,
   bool audio_streaming_allowed() const;
   void set_audio_streaming_allowed(bool state);
 
+  /**
+   * @brief set_seek_enabled Set seek enabled for application
+   * @param seek_enabled
```

Comment:
You can delete this descriptions, because they are already provided in application.h file.